### DISCUSSION TITLE
Ability to disable date and time context in history search

### DISCUSSION THREAD

**Discussion by pbadenski** [PatrickF1/fzf.fish#193] [Q&A] [answered]
I can see how a lot of people might find this feature useful - me, I prefer the absolute necessary minimum of information - otherwise I get distracted. It would be great for people like me to allow disabling history date/time info from results.

**Comment by PatrickF1** [comment, marked as answer]
https://github.com/PatrickF1/fzf.fish/wiki/Cookbook#how-can-i-disable-the-fzf-preview-pane-for-a-particular-command

**Reply by pbadenski** [reply]
Thanks, I can see how I can format a preview pane. but it's not obvious to me how I can remove date and time information from the list of results. Is that possible?

**Reply by PatrickF1** [reply]
Oops sorry I misread your question! What about this? https://github.com/PatrickF1/fzf.fish/discussions/169#discussioncomment-816956

**Reply by PatrickF1** [reply]
Ugh sorry yet again. That comment was about a related but different question. Sorry, been a hectic day so I've been reading things at 10k rpm.
Unfortunately, no, I don't think it's possible right now, sorry :(

**Reply by pbadenski** [reply]
No worries, I understand. Oh ok, in that case my idea for feature request stands :)

**Reply by PatrickF1** [reply]
Thanks for your patience. Whew, I need to stop working...
I like your feature request and can see its value, shouldn't be too hard to build but it'll have to take priority after a few other things I've been putting off for a long time.

In the meantime, would you mind giving my repo a star? I shamelessly really like getting stars haha. Np if not :)

**Reply by pbadenski** [reply]
Done! 😄

**Reply by kidonng** [reply]
Put in your Fish config:

`bind \cr 'fzf_history_opts=--with-nth=4.. _fzf_search_history'`

This overrides the default <kbd>Ctrl</kbd> + <kbd>R</kbd> shortcut and only shows the command itself.

**Reply by PatrickF1** [reply]
Thanks @kidonng! I forgot about that option. I just tested it and it seems to work. However, I think you can just do this in your config.fish
```
set fzf_history_opts --with-nth=4..
```

You shouldn't have to override the entire binding.

Pawel, can you confirm this works for you please?

**Reply by kidonng** [reply]
> However, I think you can just do this in your config.fish
> ```
> set fzf_history_opts --with-nth=4..
> ```
> 
> You shouldn't have to override the entire binding.

Because I also have `bind \e\cr _fzf_search_history`.

**Reply by PatrickF1** [reply]
Interesting, you use both versions! Could you explain why removing the timestamps is so important to you that you have a dedicated key bind for it? Collecting more user stories will help me prioritize this feature.

**Reply by pbadenski** [reply]
> Pawel, can you confirm this works for you please?

Works like a charm, thank you!!

**Reply by kidonng** [reply]
> Interesting, you use both versions! Could you explain why removing the timestamps is so important to you that you have a dedicated key bind for it?

Like the OP said, most of the time I only care about the commands themselves. That's why I override the default binding to hide the timestamps.

But there are often times I need to distinguish similar commands from a different time, or find commands in particular time, and that's when timestamps come handy.

I also change the timestamps to include the year, which may not suit as the default as they take too much space for little value. But this is not an issue for me as I don't see timestamps at all most of the time.
